tidal 1.7

```
Configuring tidal-1.7...
Preprocessing library for tidal-1.7..
Building library for tidal-1.7..
Preprocessing test suite 'dontcrash' for tidal-1.7..
Building test suite 'dontcrash' for tidal-1.7..
Preprocessing test suite 'tests' for tidal-1.7..
Building test suite 'tests' for tidal-1.7..
Preprocessing benchmark 'bench-speed' for tidal-1.7..
Building benchmark 'bench-speed' for tidal-1.7..
Preprocessing benchmark 'bench-memory' for tidal-1.7..
Building benchmark 'bench-memory' for tidal-1.7..
Preprocessing library for tidal-1.7..
Running Haddock on library for tidal-1.7..

src/Sound/Tidal/Params.hs:25:1: error:
    parse error on input ‘import’
   |
25 | import qualified Data.Map.Strict as Map
   | ^^^^^^
Haddock coverage:
   0% (  0 /  9) in 'Paths_tidal'
  Missing documentation for:
    Module header
    version (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:28)
    getBinDir (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:39)
    getLibDir (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:39)
    getDynLibDir (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:39)
    getDataDir (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:39)
    getLibexecDir (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:39)
    getDataFileName (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:47)
    getSysconfDir (.stack-work/dist/x86_64-linux/Cabal-3.2.1.0/build/autogen/Paths_tidal.hs:39)
   0% (  0 /  2) in 'Sound.Tidal.Bjorklund'
  Missing documentation for:
    Module header
    bjorklund (src/Sound/Tidal/Bjorklund.hs:44)
   0% (  0 /  3) in 'Sound.Tidal.Config'
  Missing documentation for:
    Module header
    Config (src/Sound/Tidal/Config.hs:21)
    defaultConfig (src/Sound/Tidal/Config.hs:32)
```

cc @yaxu

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -5111,9 +5111,6 @@ packages:
       # https://github.com/commercialhaskell/stackage/issues/5846
       - dbus < 1.2.18
 
-      # https://github.com/commercialhaskell/stackage/issues/5848
-      - tidal < 1.7
-
       # https://github.com/commercialhaskell/stackage/issues/5849
       - cryptonite < 0.28
 

